Clinical-Genomics/cgweb
# Validate special requirements for PAx and EXx applications in Orderportal

This PR adds the requirement of elution buffer for mip exome and panel applications 

Contact: @annagellerbring @annaengstrom 

How to install: 
1. install or ask someone to install cgweb and clinical-api branch on clinical-db

How to test validation of elution buffer:
1. create mip order in op
1. add family
1. add sample
1. select any application starting with PA or EX
1. try to save sample

Expected outcome:
Red box around elution buffer

How to test validation of capture-kit:
1. create mip order in op
1. add family
1. add sample
1. select any application starting with PA
1. try to save sample

Expected outcome:
Red box around capture kit

How to test submission of elution buffer:
1. create mip order in op
1. add family
1. add sample
1. select any application starting with PA or EX
1. select elution buffer
1. submit
1. open sample in LIMS

Expected outcome:
Elution buffer on the LIMS sample

How to test submission of capture_kit buffer:
1. create mip order in op
1. add family
1. add sample
1. select any application starting with PA 
1. select capture_kit buffer
1. submit
1. open sample in LIMS

Expected outcome:
capture_kit buffer on the LIMS sample

- [x] Tested by @barrystokman 

This is a minor **version bump** because it is a bug fix
